databricks_cli/dbfs/api.py

```diff
@@ -97,20 +107,72 @@ def file_exists(self, dbfs_path, headers=None):
             raise e
         return True
 
+    @retry(wait=wait_random_exponential(multiplier=1, max=30), 
+           retry=retry_if_exception_type(RateLimitException))
     def get_status(self, dbfs_path, headers=None):
-        json = self.client.get_status(dbfs_path.absolute_path, headers=headers)
-        return FileInfo.from_json(json)
+        try:
+            json = self.client.get_status(dbfs_path.absolute_path, headers=headers)
+            return FileInfo.from_json(json)
+        except HTTPError as e:
+            if e.response.status_code == 429:
+                click.echo("\rRate Limit Exceeded. Trying again.")
+                raise RateLimitException
+            raise e
+
+    @retry(wait=wait_random_exponential(multiplier=1, max=30), 
+           retry=retry_if_exception_type(RateLimitException))
+    def create_with_retries(self, dbfs_path, overwrite, headers):
+        try:
+            return self.client.create(dbfs_path, overwrite, headers=headers)
+        except HTTPError as e:
+            if e.response.status_code == 429:
+                click.echo("\rRate Limit Exceeded. Trying again.")
+                raise RateLimitException
```

Comment:
Same as above: we should also have `else: raise` to propagate other exceptions further.

Same for other `_with_retries` wrappers and potentially in other places that I might have missed.